main.py

```diff
@@ -71,6 +71,17 @@ def _fix_model_name(model, model_args):
     return filename
 
 
+def setup_example_logger(output_path):
+    """Sets up a logger that will save each example and prediction."""
+    logger = logging.getLogger("examples")
+    filename = f"./outputs/examples-{output_path}.jsonl"
```

Comment:
I'm getting `FileNotFoundError`s on a fresh install so you may want to import `Path`

```python
from pathlib import Path
```

and add the following here:
```python
    filename = Path(f"./outputs/examples-{output_path}.jsonl")
    if not filename.exists():
        filename.parent.mkdir(parents=True, exist_ok=True)
        filename.touch()
```

Reply:
Ahh good catch. I moved up the folder creation to the top of the script. That should fix this issue.